book/website/reproducible-research/case-studies/statistical-methods-manuscript.md

```diff
@@ -0,0 +1,146 @@
+(rr:cs:statistical-methods-manuscript)=
+## A Statistical Methods Manuscript
+
+
```

Comment:
```suggestion
## A short summary of the Manuscript

```

Reply:
Can you write a very quick summary of the paper in a few lines? What does it do, why was it written, who is the target audience of this paper? how is the knowledge transferable to other fields (this can point to the rest of the chapter)?

Thos can be added in bullet points something like this:
```
- Paper title:
- Authors:
- Cite this paper:
- Background:
- More details:
```
Let me know what you think.